lib/zinnia.js

```diff
@@ -260,7 +260,8 @@ const catchChildProcessExit = async ({
       const moduleName = capitalize(err.moduleName ?? 'Zinnia')
       const exitReason = err.exitReason ?? 'for unknown reason'
       const message = `${moduleName} crashed ${exitReason}`
-      onActivity({ type: 'error', message })
+      const type = moduleName === 'Spark' ? 'error' : 'info'
```

Comment:
alternative to consider:

```suggestion
      const type = err.moduleName === 'spark' ? 'error' : 'info'
```

Reply:
or the other way round, to make the intend more clear:

```suggestion
      const type = err.moduleName === 'voyager' ? 'info' : 'error'
```

Reply:
I like the third option most:
- When we add the third module, its crashes will be reported as errors, which makes more sense to me
- The label "Spark"/"Voyager" can change, but the module codename "spark"/"voyager" should not. Therefore, I prefer to use `err.moduleName` for the check.